bin/plugin/commands/test-plugins.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const { execSync, spawnSync } = require( 'node:child_process' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+exports.options = [];
+
+/**
+ * Command for testing all built stndalone plugins.
+ */
+exports.handler = async () => {
+	// plugin test assets.
+	const pluginTestAssets = './plugin-tests';
+
+	// Built plugins directory.
+	const builtPluginsDir = './build/';
+
+	// Buffer built plugins array.
+	let builtPlugins = [];
+
+	// root .wp-env.json file.
+	const wpEnvFile = '.wp-env.json';
+
+	// Regex to match plugins string in .wp-env.json.
+	const wpEnvPluginsRegexPattern = '"plugins": \\[(.*)\\],';
+
+	// Regex object to match wp-env plugins string.
+	const wpEnvPluginsRegex = new RegExp( wpEnvPluginsRegexPattern, 'gm' );
+
+	let wpEnvPluginsRegexReplacement = '';
+
+	// If the root .wp-env.json file is missing, abort.
+	if ( ! fs.pathExistsSync( wpEnvFile ) ) {
+		log(
+			formats.error(
+				`WP Env config file "${ wpEnvFile }" not detected in root of project.`
+			)
+		);
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
+
+	// Only try to read plugin dirs if build ir exists.
+	if ( fs.pathExistsSync( builtPluginsDir ) ) {
+		// Read all built plugins from build dir.
+		try {
+			builtPlugins = fs
+				.readdirSync( builtPluginsDir, { withFileTypes: true } )
+				.filter( ( item ) => item.isDirectory() )
+				.map( ( item ) => item.name );
+		} catch ( e ) {
+			log(
+				formats.success(
+					`Unable to read plugins from ${ builtPluginsDir }: ${ e }`
+				)
+			);
+		}
+	} else {
+		log(
+			formats.error(
+				`Built plugins directory at ${ builtPluginsDir } does not exist. Please run the 'npm run build-plugins' command first.`
+			)
+		);
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
+
+	// For each built plugin, copy the test.
+	builtPlugins.forEach( ( plugin ) => {
+		log(
+			formats.success(
+				`Detected built plugin "${ plugin }", copying test files for standalone/integration testing.`
+			)
+		);
+
+		// Copy over test files.
+		try {
+			fs.copySync( pluginTestAssets, `./build/${ plugin }/`, {
```

Comment:
```suggestion
			fs.copySync( pluginTestAssets, `${ builtPluginsDir }${ plugin }/`, {
```

Use already define variable.